src/main/java/com/urswolfer/gerrit/client/rest/http/changes/ListChangesOptionByVersion.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2020 GerritForge Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.urswolfer.gerrit.client.rest.http.changes;
+
+import com.google.gerrit.extensions.client.ListChangesOption;
+
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+class ListChangesOptionByVersion {
+  static final Map<String, ListChangesOption> maxChangeOptionByVersion = new HashMap<String, ListChangesOption>();
+
+  static {
+    put("2.6", ListChangesOption.DETAILED_ACCOUNTS);
+    put("2.7", ListChangesOption.MESSAGES);
+    put("2.8", ListChangesOption.DOWNLOAD_COMMANDS);
+    put("2.9", ListChangesOption.DOWNLOAD_COMMANDS);
```

Comment:
This is also listed in 2.8 - correct?

Reply:
Yep, they have both the same constraint. When the version isn't listed, it means that supports all the options.